plugins/api/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import PluginRegistry from './plugin';
+
+const registry = PluginRegistry.getInstance();
+
+const __experimental = {
+	registerPlugin: registry.registerPlugin,
+	getRegisteredUIComponent: registry.getRegisteredUIComponent,
```

Comment:
Do we have no way to just retrieve the settings of a plugin? A `getPlugin` method is what I'd expect, at which point a `getRegisteredUIComponent` may not even be necessary.